@here/harp-omv-datasource/test/MapViewPickingTest.ts

```diff
@@ -391,8 +404,42 @@ describe("MapView Picking", async function() {
                           testCase.lookAt.length > 2 ? testCase.lookAt[2] : undefined
                       )
                     : rayOrigin;
+                // typeof required because the bind returns any
+                const getTile: typeof geoJsonDataSource.getTile = geoJsonDataSource.getTile.bind(
+                    geoJsonDataSource
+                );
```

Comment:
actually this was the part you could get rid of now that you use sinon, since sinon saves for you the original function. anyway, it's minor.

Reply:
you mean using `callThrough()`, awesome, didn't know this was possible, thanks.

Reply:
I tried `const tile = stub.callThrough()(_tileKey, delayLoad);`, however this gives infinite recursion, how do I get access to the original method? I couldn't find anything here: https://sinonjs.org/releases/latest/stubs/

Reply:
Thanks, didn't know this!